tsdb/engine/tsm1/wal.go

```diff
@@ -285,6 +288,10 @@ func (l *Log) addToCache(points []models.Point, fields map[string]*tsdb.Measurem
 	for _, p := range points {
 		for name, value := range p.Fields() {
 			k := SeriesFieldKey(string(p.Key()), name)
+			if len(k) > maxKeySize {
```

Comment:
because I have to encode its length and I use a `uint16` for it. 65k should be enough for anyone, no?